docs/guided-projects/advanced/configuring-sn-labs-tool-config.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+---
+# Configuring the Skills Network Labs Tool Configuration
```

Comment:
```suggestion
# Using a Custom Outline
```

Want to keep the titles short so the sidebar is readable

Also the only people reading this guide will be the people who want to deviate from the default "simplified" outline, so I think this title frames the content better